static/docs/commands-reference/version.md

```diff
@@ -0,0 +1,41 @@
+# version
+
+This command shows the system/environment information along with the DVC version.
```

Comment:
81 symbol

Reply:
@shcheklein Is there a  VSCode extension for this?

Reply:
yep, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) - just select a paragraph and hit Alt-Q. I also set vertical rulers - 80 and 100 symbols. + highlighting for the trailing spaces.